app/components/captureTasksController.js

```diff
@@ -0,0 +1,24 @@
+app.controller('CaptureTasksController', [function(){
+  var self = this
+  this.tasks = []
+  this.taskFilter = {}
+  this.addTask = function(task){
+    self.tasks.unshift({'desc': task,'createdAt': Date.now(),'completed': false});
```

Comment:
Use something a little more abstract than `desc` for targeting instances - so an `id` or something